doc/gobstones.tex

```diff
@@ -0,0 +1,146 @@
+\documentclass{article}
```

Comment:
¿Podremos [migrar esto a Markdown](https://codedocean.wordpress.com/2014/02/05/converting-latex-to-markdown/) y ponerlo en el `README.md` del repo?